src/Components/RegistrationProgress/RegAssistantFooter.js

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Icon, Text, TextContent, TextVariants } from '@patternfly/react-core';
 import { monitoringHostsLink, remoteHostConfigLink } from '../../constants';
 import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 
-const RegAssistantFooter = () => {
+const RegAssistantFooter = ({ operatingSystem }) => {
   return (
     <TextContent style={{ marginTop: '24px' }}>
-      <Text component={TextVariants.p}>
-        Read more about Remote host configuration (RHC) options and levels of
-        connectivity:{' '}
-        <a
-          rel="noopener noreferrer"
-          target="_blank"
-          href={remoteHostConfigLink}
-        >
-          Remote Host Configuration and Management | Red Hat Product
-          Documentation{' '}
-          <Icon className="pf-v5-u-ml-xs">
-            <ExternalLinkAltIcon />
-          </Icon>
-        </a>
-      </Text>
+      {(operatingSystem.id === 'centos-radio' ||
+        operatingSystem.id === 'rhel9-radio') && (
```

Comment:
If we can not avoid hardcoding this, we should at least extract these strings and put them into a descriptive constant.